build.rs

```diff
@@ -0,0 +1,9 @@
+extern crate autocfg;
+
+fn main() {
+    //let ac = autocfg::new();
+    //ac.emit_sysroot_crate("std");
+    //ac.emit_sysroot_crate("alloc");
+    autocfg::emit("std");
```

Comment:
What does this do? Commit logs don't explain it.

Reply:
That would blindly [emit](https://docs.rs/autocfg/0.1.6/autocfg/fn.emit.html) a `std` config value. I suppose they were doing that as a stopgap until the commented crate tests were available -- that's now in autocfg 0.1.6.